docs/sql-data-sources-csv.md

```diff
@@ -108,6 +108,12 @@ Data source options of CSV can be set via:
     <td>Infers the input schema automatically from data. It requires one extra pass over the data. CSV built-in functions ignore this option.</td>
     <td>read</td>
   </tr>
+  <tr>
+    <td><code>inferDate</code></td> 
+    <td>false</td>
+    <td>Whether or not to infer columns that satisfy the <code>dateFormat</code> option as <code>Date</code>. Requires <code>inferSchema</code> to be <code>true</code>. When <code>false</code>, columns with dates will be inferred as <code>String</code> (or as <code>Timestamp</code> if it fits the <code>timestampFormat</code>) Legacy date formats in Timestamp columns cannot be parsed with this option.</td>
```

Comment:
By the way, what does this mean?

>Legacy date formats in Timestamp columns cannot be parsed with this option.

Reply:
This was in reference to the fact that only the non-legacy datePaser is used for parsing in timestamp columns - as explained in [the first comment of this thread](https://github.com/apache/spark/pull/36871/files#r903187641).

That sentence no longer makes sense since we don't support date inference for legacy date formats.